services/forum/src/main.py

```
@@ -243,6 +243,16 @@ def list_my_subscriptions(
     return logic.get_subscriptions_by_user(db, user_id)
 
 
+@router.get("/projects/users/{user_id}/subscriptions", response_model=List[schemas.ProjectResponse])
+def list_user_subscriptions(
+    user_id: int,
+    db: Session = Depends(get_db),
```

Ruff (0.15.10)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)